src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "title": "Static Analysis Results Format (SARIF) Version 2.0.0 JSON Schema",
-  "description": "Static Analysis Results Format (SARIF) Version 2.0.0 JSON Schema: a standard format for the output of static analysis tools.",
+  "title": "Static Analysis Results Format (SARIF) Version 2.0.0-beta.2018-09-26 JSON Schema",
```

Comment:
>beta [](start = 65, length = 4)

csd2 might be a better term here:

2.0.0-csd2.2018-09-26

or perhaps add both for clearest communication?

2.0.0-beta.csd2.2018-09-26

Reply:
That sounds good. Be aware that according to SemVer the, pre-release version is supposed to sort chronologically (so that, for example, `alpha.1` < `alpha.2` < `beta.1`. Whether we stay with what Jim proposed, adopt one of your two suggestions, or go with `csd2.beta.YYYY-MM-DD` (which I like better because "beta" qualifies "CSD.2" -- these are beta versions of CSD.2(, we will break that rule.

I'm going to go with `2.0.0-csd.2.beta.YYYY-MM-DD`.

---
In reply to: [221751980](https://github.com/Microsoft/sarif-sdk/pull/1105#discussion_r221751980) [](ancestors = 221751980)

Reply:
Fixed.

---
In reply to: [221758555](https://github.com/Microsoft/sarif-sdk/pull/1105#discussion_r221758555) [](ancestors = 221758555,221751980)